config/SDKVersions.xcconfig

```diff
@@ -1,10 +0,0 @@
-_WP_IOS_130000 = XCODE11
-_WP_IOS_120000 = XCODE10
-
-WP_IOS_SDK = $(_WP_IOS_$(SDK_VERSION_MAJOR))
-
-SWIFT_ACTIVE_COMPILATION_CONDITIONS = $(inherited) $(WP_IOS_SDK)
```

Comment:
⚠️  Please note, I have removed this build setting since we don't use Xcode 11 nor 10.

Reply:
Excellent! I was looking around those files recently and was wondering what the point of them was.

As far as I can see, `WP_IOS_` is unused in the codebase. We should be good 👍